manifest/openshift4/coreos/ncp-openshift4.yaml

```diff
@@ -891,8 +882,15 @@ spec:
         # - name: dir-tmp-usr-ovs-kmod-backup
         #   hostPath:
         #     path: /tmp/nsx_usr_ovs_kmod_backup
-
-
+{{end}}
+{{if .UseNsxOvsKmod}}
+        - name: host-usr-share
+          hostPath:
+            path: /usr/share
+        - name: host-depmod
+          hostPath:
+            path: /sbin/depmod
+{{else}}
```

Comment:
we can remove code line #L893 - 899.
{{else}}
..
#path: /sbin/depmod

Reply:
Hello Yun, the {{else}} part and the comments has been removed

Reply:
Thanks shizhao.

one more comment: I see you add 6 commits in this PR, could squash these 6 commits into one? 
Usually, we just use git commit --amend to squash multiple commits into one, so that we could have 
a clear merge tree in main branch. Thanks